### DISCUSSION TITLE
Request for IME Support for Non-Latin Languages, Especially Japanese

### DISCUSSION THREAD

**Discussion by GameHintDev** [GitbookIO/community#435] [Feature Requests]
Hello,

I've been using GitBook and noticed some issues with the Input Method Editors (IMEs) for non-Latin languages, like Japanese, that I wanted to share with you.

Here are the problems:
1. When I type the page title and press enter, the same text gets copied into the page description.
2. Typing content overlaps with the placeholder text, making it hard to read.
3. Trying to use tab completion while typing automatically inputs the current characters.

These issues are making it tough for Japanese users, and probably for those using other non-Latin languages too. Considering GitBook's global user base, improving IME support for languages including Japanese seems really important for user-friendliness.

Hoping to see these issues fixed in a future update.

https://github.com/GitbookIO/community/assets/64947947/60d05f7c-49e6-4858-a9e1-ed19d1685dc6

**Comment by KohjiNakahata** [comment]
I recently transitioned to using GitBook as well, and I am facing similar issues.
I am satisfied with the other features and would like to continue using it, so I would greatly appreciate it if you could address this issue.

**Comment by setoyama-precena** [comment]
We  manage our company’s technical information site on Gitbook.com. However, we're having trouble with the input experience when using Japanese, especially with text conversion, which is quite inconvenient.

https://tech-book.precena.co.jp/

We understand that there might not be many Japanese users, but it would be very helpful if the priority for this issue could be increased.
